Amethyst/Layout/ThreeColumnLayout.swift

```diff
@@ -0,0 +1,237 @@
+//
+//  ThreeColumnLayout.swift
+//  Amethyst
+//
+//  Originally created by Ian Ynda-Hummel on 12/15/15.
+//  Copyright © 2015 Ian Ynda-Hummel. All rights reserved.
+//
+//  Modifications by Craig Disselkoen on 08/31/18.
+//
+
+import Silica
+
+enum WhichColumn {
```

Comment:
Just `Column` is better than `WhichColumn`. You could nest it under the Layout definition if you're concerned about namespace, but this might be able to be `private` as the behavior seems to be encapsulated by the three layout class definitions, so nothing need concern itself with the details of these enums.

Reply:
`private` seems like the best solution to me.  Thanks for the suggestion!